Tasks/AzureTestPlanV0/testLibExecutor.ts

```diff
@@ -0,0 +1,91 @@
+import os = require('os');
+import Q = require('q');
+import path = require('path');
+import fs = require('fs');
+import * as tl from 'azure-pipelines-task-lib/task';
+import * as tr from 'azure-pipelines-task-lib/toolrunner';
+
+var isWindows = os.type().match(/^Win/);
+var m2HomeEnvVar: string = null;
+var mavenPOMFile: string = tl.getPathInput('mavenPOMFile', true, true);
```

Comment:
since we will be adding gradle and python execution in same file, we should move this input of maven inside maven method

Reply:
Updated